backend/app/api/endpoints/airflow.py

```diff
@@ -1,99 +0,0 @@
-import json
-from datetime import datetime,timedelta
```

Comment:
Removing this endpoint will get an error in current dashboard

Reply:
oh, this is probably a weird thing with Git, I moved the file inside /v1/endpoints

Reply:
Ack